challenges/functions.js

```diff
@@ -6,17 +6,28 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+let  Func = (param1,param2,cb) => {
+  return cb(param1 + param2)
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+let func = (param1, param2, cb) => {
+  return cb(param1 + param2)
+}
 
+let tuck = (param1, param2, cb) => {
+return cb(param1 * param2)
+}
+let greeting = (first,last) => {
+  return `Hello ${this.first} ${this.last}, nice to meet you!`
```

Comment:
* The use of this inside the function is not necessary, since you have direct access to first and last variables.
* Consider doing it like so: return `Hello ${first} ${last}, nice to meet you!`